### PR TITLE
Release v1.3.4

### PR DESCRIPTION
**Pull request by bryanculver**
Request to merge `develop` into `main` to release v1.3.4.

# TODO

- [x] Update pyproject.toml 
- [x] Update date on changelog

### CODE REVIEW

File: nautobot/docs/configuration/optional-settings.md [hunk 1]
```diff
@@ -156,6 +156,14 @@ If a custom URL is not provided for any of the links, the default link within th
...
 ---
 
+## BRANDING_PREPENDED_FILENAME
+
+Default: `"nautobot_"`
+
+Environment Variable: `NAUTOBOT_BRANDING_PREPENDED_FILENAME`
+
+Defines the initial pre-pended portion of a filename for for export to CSV/YAML or export templates.
+
```
Comment (glennmatthews) [inline nautobot/docs/configuration/optional-settings.md:166]:
```suggestion
Defines the prefix of the filename when exporting to CSV/YAML or export templates.

---

```

File: nautobot/docs/rest-api/authentication.md [hunk 2]
```diff
@@ -40,6 +40,8 @@ $ curl http://nautobot/api/dcim/sites/
 
 ## Initial Token Provisioning
 
+_Added in version [1.3.0](../release-notes/version-1.3.md/#rest-api-token-provisioning-1374)_
```
Comment (glennmatthews) [inline nautobot/docs/rest-api/authentication.md:43]: Do we have a housekeeping item open for going through and adding this sort of note throughout?

File: nautobot/docs/configuration/optional-settings.md [hunk 3]
```diff
@@ -815,6 +823,21 @@ This parameter defines the URL of the repository that will be checked periodical
 
 ---
 
+## SANITIZER_PATTERNS
+
```
Comment (glennmatthews) [inline nautobot/docs/configuration/optional-settings.md:827]:
```suggestion

_Added in version 1.3.4_

```

File: nautobot/docs/configuration/optional-settings.md [hunk 4]
```diff
@@ -156,6 +156,14 @@ If a custom URL is not provided for any of the links, the default link within th
 
 ---
 
+## BRANDING_PREPENDED_FILENAME
+
```
Comment (glennmatthews) [inline nautobot/docs/configuration/optional-settings.md:160]:
```suggestion

_Added in version 1.3.4_

```

File: nautobot/docs/rest-api/authentication.md (hunk 2 above)
Comment (bryanculver) [reply nautobot/docs/rest-api/authentication.md:43]: https://github.com/nautobot/nautobot/issues/1778

File: pyproject.toml [hunk 5]
```diff
@@ -2,7 +2,7 @@
 name = "nautobot"
 # Primary package version gets set here. This is used for publishing, and once
 # installed, `nautobot.__version__` will have this version number.
-version = "1.3.3"
+version = "1.3.4-beta.1"
```
Comment (bryanculver) [inline pyproject.toml:5]:
```suggestion
version = "1.3.4"
```

**Review by glennmatthews — APPROVED** [review]

**Pull request merged**